package.json

```diff
@@ -74,7 +74,7 @@
     "eslint-plugin-jest": "^21.6.1",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.5.1",
-    "flow-bin": "0.61.0",
+    "flow-bin": "0.67.1",
```

Comment:
0.68 is released, FWIW

Reply:
@SimenB I had some problems with it. Gonna try again.